apps/openmw/mwgui/inventorywindow.cpp

```diff
@@ -1,7 +1,5 @@
 #include "inventorywindow.hpp"
 
-#include <stdexcept>
```

Comment:
Why did you remove this include? `<stdexcept>` is one that haunts me in particular because apparently some versions of GCC implicitely include `<stdexcept>` from some other header but other versions don't, leading to easy-to-miss errors that only pop up later breaking the build for a subset of platforms.

Used on line 316:
<pre>
throw std::runtime_error("Can't find restacked item");
</pre>